articles/azure-functions/functions-bindings-storage-table.md

```diff
@@ -417,7 +417,7 @@ The following example shows an HTTP triggered function which returns the total c
 @FunctionName("getallcount")
 public int run(
    @HttpTrigger(name = "req",
-                 methods = {"get"},
+                 methods = {"HttpMethod.GET"},
```

Comment:
This should be without the double quotes it now accepts an HttpMethod, not String.
```suggestion
                 methods = {HttpMethod.GET},
```